docs/decisions/0015-s3gw-ha-model.md

```diff
@@ -0,0 +1,31 @@
+# s3gw ha model
+
+## Context and Problem Statement
+
+We analyzed some High Availability - HA - concepts applied to s3gw when used with Longhorn.
+The final aim of the research is to identify an HA model we can use with the s3gw.
+
+The full HA research work conducted until now has been proposed with the
+[High Availability research](https://github.com/aquarist-labs/s3gw/pull/685) Pull request.
+You can find there all the rationales, motivations and the details about the tests performed.
+
+## Considered Options
+
+We identified 3 HA models:
+
+- *Active/Active*
+- *Active/Warm Standby*
+- *Active/Standby*
+
+## Decision Outcome
+
+The 3 HA models have different performances and different implementation efforts.
+For our use case, the *Active/Standby* model built on top of Longhorn actually makes
+the most sense and brings the "best" HA characteristics relative to implementing a
+more fully active/distributed solution.
+
+The *Active/Standby* model is expected to work with nothing but the primitives normally
+available on any Kubernetes cluster.
+
+for a more comprehensive explanation about this choice, please refer to the
+[High Availability research](https://github.com/aquarist-labs/s3gw/pull/685) Pull request.
```

Comment:
for consistency, I'd either capitalize "Request", or write "pull request" instead.

Reply:
For the time being I will go with all downcase; anyway, I'm not sure we should point the PR; perhaps better link the HA document once merged?